src/main.rs

```diff
@@ -411,13 +411,15 @@ async fn main() -> Result<()> {
 
     // Drop privileges
     if let Some(user_group) = &config.run_as {
-        permissions::chown(user_group, &mount);
+        permissions::chown(user_group, &mount)
+            .expect("Error when changing directory ownership");
```

Comment:
```suggestion
        permissions::chown(user_group, &mount)?;
```
In this case it may be slightly clearer to just return the error from `permissions::chown` directly, as it will print out the reason it failed and the path.

Reply:
Yes, done!